docs/getting-started/linux-installer.md

```diff
@@ -1,391 +1,5 @@
-> The official Splunk documentation for this page is [Install on Linux](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html). For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING#documentation.md).
+The official Splunk documentation for this page is [Install the Collector for Linux](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html). 
 
-# Linux Installer Script
+For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING#documentation.md).
```

Comment:
Can we hold off on all installer docs until and audit has been done to confirm all the info exists in the product docs? I believe these are frequently linked and omissions would be undesirable.

Reply:
Of course!

Reply:
To clarify, this includes all the `*-installer` and `*-manual` content. Adding a migration notice and link to the corresponding product doc could be an intermediary step.